pages/linux/pidstat.md

```diff
@@ -0,0 +1,23 @@
+# pidstat
+
+> Show system resource usage, including CPU, memory, IO etc.
+
+- Show CPU statistics at a given interval and a given number of times:
+
+`pidstat {{interval}} {{times}}`
```

Comment:
Let us use concrete values for "interval" and "times", and update the description to match those values.